### PR TITLE
Companion PR for Bounties #5715

### PR DESCRIPTION
**Pull request by xlc**
Companion PR for paritytech/substrate#5715

### CODE REVIEW

File: runtime/polkadot/src/lib.rs [hunk 1]
```diff
@@ -524,7 +524,12 @@ parameter_types! {
 	pub const TipCountdown: BlockNumber = 1 * DAYS;
 	pub const TipFindersFee: Percent = Percent::from_percent(20);
 	pub const TipReportDepositBase: Balance = 1 * DOLLARS;
-	pub const TipReportDepositPerByte: Balance = 1 * CENTS;
+	pub const DataDepositPerByte: Balance = 1 * CENTS;
+	pub const BountyDepositBase: Balance = 1 * DOLLARS;
+	pub const BountyDepositPayoutDelay: BlockNumber = 7 * DAYS;
+	pub const BountyValueMinimum: Balance = 20 * DOLLARS;
+	pub const BountyDuration: BlockNumber = 14 * DAYS;
+	pub const MaximumReasonLenght: u32 = 16384;
```
Comment (rphmeier) [inline runtime/polkadot/src/lib.rs:532]: typo, should be `Length`.

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -455,7 +455,12 @@ parameter_types! {
 	pub const TipCountdown: BlockNumber = 1 * DAYS;
 	pub const TipFindersFee: Percent = Percent::from_percent(20);
 	pub const TipReportDepositBase: Balance = 1 * DOLLARS;
-	pub const TipReportDepositPerByte: Balance = 1 * CENTS;
+	pub const DataDepositPerByte: Balance = 1 * CENTS;
+	pub const BountyDepositBase: Balance = 1 * DOLLARS;
+	pub const BountyDepositPayoutDelay: BlockNumber = 3 * DAYS;
+	pub const BountyValueMinimum: Balance = 20 * DOLLARS;
+	pub const BountyDuration: BlockNumber = 14 * DAYS;
+	pub const MaximumReasonLenght: u32 = 16384;
```
Comment (rphmeier) [inline runtime/kusama/src/lib.rs:463]: I don't have strong opinions on these constants but it seems like something that should bear further review.

cc @gavofyork @rrtti

**Comment by xlc** [comment]
Update the parameters so they makes more sense with current token price.

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**